widgetssdk/src/main/java/com/glia/widgets/call/CallView.kt

```diff
@@ -970,7 +971,9 @@ internal class CallView(
                 callState.callStatus.formattedOperatorName,
                 ""
             )
-            if (callState.companyName != null) {
+            if (callState.companyName != null) { // TODO FIXME before release
```

Comment:
What exactly should be fixed within this TODO?

Reply:
I think it makes sense to be more specific because for me, at least it's not obvious here...

Reply:
The company name should be configurable from remote strings, so we need to have a way of using the companyName from remote strings instead of the one from GliaInit